src/databricks/labs/lakebridge/resources/assessments/mssql/common/connector.py

```diff
@@ -0,0 +1,22 @@
+from databricks.labs.lakebridge.connections.database_manager import DatabaseManager
+
+
+def get_sqlserver_reader(
```

Comment:
Database manager should already have the MSSQLConnector we should be using that.